packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -488,7 +488,10 @@ class RenderParagraph extends RenderBox
     _textPainter.markNeedsLayout();
   }
 
+  List<PlaceholderDimensions> _placeholderDimensions;
```

Comment:
Might be worth adding non-public documentation explaining why this is needed.

Reply:
done